client/src/test/java/org/apache/celeborn/client/ShuffleClientSuiteJ.java

```diff
@@ -105,11 +105,14 @@ public void testPushData() throws IOException, InterruptedException {
               1,
               1);
 
-      Compressor compressor = Compressor.getCompressor(conf);
-      compressor.compress(TEST_BUF1, 0, TEST_BUF1.length);
-      final int compressedTotalSize = compressor.getCompressedTotalSize();
-
-      assert (pushDataLen == compressedTotalSize + BATCH_HEADER_SIZE);
+      int totalSize = TEST_BUF1.length;
+      if (!codec.equals(CompressionCodec.NONE)) {
+        Compressor compressor = Compressor.getCompressor(conf);
+        compressor.compress(TEST_BUF1, 0, TEST_BUF1.length);
+        totalSize = compressor.getCompressedTotalSize();
+      }
+
+      assert (pushDataLen == totalSize + BATCH_HEADER_SIZE);
```

Comment:
```suggestion
      ```suggestion
      if (codec.equals(CompressionCodec.NONE)) {
        assert (pushDataLen == TEST_BUF1.length + BATCH_HEADER_SIZE);;
      } else {
        Compressor compressor = Compressor.getCompressor(conf);
        compressor.compress(TEST_BUF1, 0, TEST_BUF1.length);
        final int compressedTotalSize = compressor.getCompressedTotalSize();
        assert (pushDataLen == compressedTotalSize + BATCH_HEADER_SIZE);
      }
```
```

Reply:
@pan3793 ok.

Reply:
@pan3793 done.